src/app/users/users.component.ts

```diff
@@ -1,15 +1,22 @@
+import { HttpClient } from '@angular/common/http';
 import { Component, OnInit } from '@angular/core';
+import { IUser } from './user.interface';
 
 @Component({
   selector: 'app-users',
   templateUrl: './users.component.html',
-  styleUrls: ['./users.component.css']
+  styleUrls: ['./users.component.css'],
```

Comment:
use  scss ,  configure angular to use scss .

Reply:
ok !!

Reply:
done thanks